### DISCUSSION TITLE
Less harsh dark mode default/clearer background color customisation

### DISCUSSION THREAD

**Discussion by nikspyratos** [filamentphp/filament#7350] [Ideas]
Loving V3!

Honestly, my only gripe is the harsh dark mode. Tailwind's `bg-gray-950` is a particularly dark shade, which makes for a large contrast. Not the best on the eyes. `bg-gray-50` for light mode is better since it isn't straight `#FFFFFF`.

I played around with the colours and noticed that it's the `gray` color setting that dictates this for both light and dark modes.

With that, the best way to do this right now would be to override the `950` value of `Color::Gray`, which feels a bit unintuitive. 

Of course, once you get into full theme customisation this is less of an issue as you'd likely pick a different color.

Perhaps this should be split into a `background` color that uses a smaller range of values for the light and dark modes? Or alternatively, use the `800` value for the main background and either `700` or `900` for the top nav/panels.

https://www.webmd.com/eye-health/contrast-sensitivity-issues-with-vision
https://www.healthline.com/health/is-dark-mode-better-for-your-eyes

**Comment by danharrin** [comment]
We're looking at releasing themes with lighter dark modes 👍
